dotnet/src/Connectors/Connectors.GoogleVertexAI/Core/Gemini/GeminiRequest.cs

```diff
@@ -0,0 +1,247 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.ChatCompletion;
+
+namespace Microsoft.SemanticKernel.Connectors.GoogleVertexAI.Core;
+
+internal sealed class GeminiRequest
```

Comment:
We should consider adding all of the internal model classes used by the client into a `Models` folder. Namespace can remain the same.

Reply:
Issue #5676 created to track this suggestion.